src/analysis/rule_extraction.jl

```diff
@@ -103,18 +109,21 @@ function print_rules(rules::Vector{Rule{Vector{Vector},Vector{Float64}}})::Nothi
 end
 
 """
-    cluster_rules(clusters::Vector{T}, X::DataFrame, max_rules::T; seed::Int64=123, kwargs...) where {T<:Integer,F<:Real}
-    cluster_rules(clusters::Union{BitVector,Vector{Bool}}, X::DataFrame, max_rules::T; kwargs...) where {T<:Int64}
+    cluster_rules(clusters::Vector{T}, X::DataFrame, max_rules::T;seed::Int64=123, remove_duplicates::Bool=true, kwargs...)::Vector{Rule{Vector{Vector},Vector{Float64}}} where {T<:Int64}
+    cluster_rules(clusters::Union{BitVector,Vector{Bool}}, X::DataFrame, max_rules::T;seed::Int64=123, remove_duplicates::Bool=true, kwargs...)::Vector{Rule{Vector{Vector},Vector{Float64}}} where {T<:Int64}
 
 Use SIRUS package to extract rules from time series clusters based on some summary metric
 (default is median). More information about the keyword arguments accepeted can be found in
 MLJ's doc (https://juliaai.github.io/MLJ.jl/dev/models/StableRulesClassifier_SIRUS/).
 
 # Arguments
-- `clusters` : Vector of cluster indexes for each scenario outcome
-- `X` : Features to be used as input by SIRUS
-- `max_rules` : Maximum number of rules, to be used as input by SIRUS
-- `seed` : Seed to be used by RGN
+- `clusters` : Vector of cluster indexes for each scenario outcome.
+- `X` : Features to be used as input by SIRUS.
+- `max_rules` : Maximum number of rules, to be used as input by SIRUS.
+- `seed` : Seed to be used by RGN. Defaults to 123.
```

Comment:
Okay for now, just remember the conversation about a user-defined or global RNG

Reply:
I did that before that conversation. I don't mind changing this, I really think your way is better.

Reply:
Leave as is for now - this is good enough.